microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -10,59 +10,65 @@ ms.date: 08/03/2021
+#### 3. Run forward-compatibility tests when using the Evergreen Runtime.
 
-To solve this situation, before your code calls a recently added WebView2 API, test whether that API is available in the client's installed runtime.  This test for newer functionality is similar to other web development best practices that detect supported features before using new web APIs.  To test for API availability in the installed runtime, use either:
+As Evergreen Runtime updates automatically, you should regularly run compatibility tests to ensure that your WebView2 app will continue to work as expected on the new runtime version. This can be done by testing your web content in the WebView2 control against the Microsoft Edge preview channels (Beta, Dev, or Canary).  
 
-*  `QueryInterface` in C/C++.
-*  A `try/catch` block in .NET or WinUI.
+We recommend following the guidance in [Prerelease testing using preview channels](../how-to/prerelease-testing.md) and [Self-host by deploying preview channels](../how-to/self-hosting.md).
 
-See [Feature-detecting to test whether the installed Runtime supports recently added APIs](../concepts/versioning.md#feature-detecting-to-test-whether-the-installed-runtime-supports-recently-added-apis).
+#### 4. Use feature-detection to test whether the installed Runtime supports recently added APIs.
 
+To run a WebView2 app that was developed with a particular version of the Webview2 SDK, the client must have a compatible version of the WebView2 Runtime installed. 
 
-<!-- ====================================================================== -->
-## Update the Fixed Version Runtime
+When using the Evergreen WebView2 Runtime, there are some scenarios where the runtime on a client hasn't been automatically updated to the latest version. Additionally, some group policies pause updating of the runtime.  As such, when you push an update to your WebView2 app, the app might not work if it tries to call newer APIs that aren't available in the client's installed runtime.
 
-If you use the Fixed Version WebView2 Runtime, make sure you regularly update the WebView2 Runtime that's packaged with your app, to reduce security risks.  When using third-party content in Webview2 apps, always consider the content to be untrusted.  See [Fixed Version distribution mode](../concepts/distribution.md#details-about-the-fixed-version-runtime-distribution-mode).
+Use feature-detection to make sure that the newer APIs that are used by your WebView2 app are supported by the WebView2 Runtime that's installed on the client. See [Feature-detecting to test whether the installed Runtime supports recently added APIs](../concepts/versioning.md#feature-detecting-to-test-whether-the-installed-runtime-supports-recently-added-apis).
 
 
 <!-- ====================================================================== -->
-## Manage new versions of the Evergreen Runtime
+### Update regularly if using the Fixed Version Runtime
```

Comment:
keep or delete section? "Update regularly if using the Fixed Version Runtime"

Reply:
@liminzhu 

As mentioned, - since from our POV, fixed version isn't the best practice per-say, it's confusing to keep this note here. 
I am inclined to remove this section.

Reply:
rendered article section: 
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/developer-guide?branch=pr-en-us-3195#update-regularly-if-using-the-fixed-version-runtime

Reply:
I would keep the section actually. Evergreen is recommended for most developers but there are legitimate use cases for fixed versions and it is a mode that we support. Our best practice is not you should not use fixed version. 

For developers using fixed version, the best practice is that you should consider updating regularly.